src/SmartComponents/CreatePolicy/CreatePolicy.js

```diff
@@ -16,9 +16,16 @@ export const CreatePolicy = ({
     benchmark, osMajorVersion, complianceThreshold, name, profile, refId, selectedRuleRefIds, systemIds
 }) => {
     const history = useHistory();
+    const location = useLocation();
     const [stepIdReached, setStepIdReached] = useState(1);
+
+    const resetAnchor = () => {
+        history.push({ state: location.state, hash: '' });
```

Comment:
Are we sure `state` and `hash` are the only things required here? We could play it safer and spread everything in:

```suggestion
        history.push({ ...location, hash: '' });
```